src/e2e/step-definitions/click.ts

```diff
@@ -28,6 +34,15 @@ When(/^I click the "([^"]*)" (?:button|link|icon|element|radio button) "([0-9])"
   }
 });
 
+// multiple times click
+When(/^I click the "(1st|2nd|3rd|[0-9]+th)" "([^"]*)" (?:button|link|icon|element) "([^"]*)" times$/, async (elementIndex: string, elementName: string, elementCount: number) => {
+  const index = parseInt(elementIndex, 10) - 1;
```

Comment:
Maybe add error handling for `0th` ? (Not sure what `getElementByCss` would do in case of a negative index...